cmd/pipeline/destroy.go

```diff
@@ -89,6 +91,7 @@ okteto pipeline destroy --wait=false`,
 	cmd.Flags().BoolVarP(&flags.wait, "wait", "w", true, "wait until the Development Environment is destroyed")
 	cmd.Flags().BoolVarP(&flags.destroyVolumes, "volumes", "v", false, "destroy persistent volumes created by the Development Environment")
 	cmd.Flags().DurationVarP(&flags.timeout, "timeout", "t", fiveMinutes, "the duration to wait for the Development Environment to be destroyed. Any value should contain a corresponding time unit e.g. 1s, 2m, 3h")
+	cmd.Flags().BoolVar(&flags.destroyDependencies, "dependencies", false, "destroy repositories in the 'dependencies' section")
```

Comment:
Should we use the same description in all the `--dependencies` flag of destroy commands? The one in `okteto destroy` has another description